pkg/rtc/rtc.go

```diff
@@ -11,6 +11,19 @@ import (
 
 type RTC struct {
 	file *os.File
+	syscalls
+}
+
+type RTCTime struct {
```

Comment:
This whole struct can be replaced by `unix.RTCTime` in every usage since it's only referenced by the _linux.go files now.